docs/index.md

```diff
@@ -179,6 +179,78 @@ Instead of using one of the opinionated `Fixture`s, you can extend the unopinion
 
 <<< @/../examples/transformer.test.ts#example
 
+## Migration Guide
+
+### v1 to v2
+
+The v2 version is a total rewrite of v1.
+Thanks for all the help [@THEtheChad](https://twitter.com/thethechad) 🤝
+
+#### Why a rewrite?
+
+v1 was flexible and allowed that multiple validation libraries could be supported in the future.
+But, this made things more complex and I don't think we intended to add more libraries than `zod`.
+
+v2 is a full-on `zod` version.
+This benefits you because we make more use of zod's schema while creating fixtures.
+For example, when you want to create a custom generator (previously a customization) you can also access zod's schema definition.
+
+> Fixture Generation with 1:1 Zod Parity
+
+#### Breaking changes
+
+##### createFixture
+
+`createFixture` still exists, but it could be that it generated its output with a slightly different output.
+It still is compatible (even more compatible) with zod's schema.
+For example, the changes to a string output:
+
+BEFORE:
+
+```
+street-a088e991-896e-458c-bbbd-7045cd880879
+```
+
+AFTER:
+
+```
+fbmiabahyvsy-vm
+```
+
+##### Customization
+
+`Customization` is renamed to `Generator`.
```

Comment:
We should also link to extending a fixture, because you no longer pass customizations (IE Generators) to `createFixture`.

Reply:
These comments should be resolved.